.github/workflows/build-test-package.yml

```diff
@@ -37,11 +37,12 @@ jobs:
           sudo apt install openslide-tools python3-openslide
           python -m pip install --upgrade pip setuptools wheel
           pip install 'flake8${{ matrix.flake8-python-git-tag }}' 'pooch${{ matrix.pooch-python-git-tag }}' 'pytest${{ matrix.pytest-python-git-tag }}'
-          pip install 'large-image[bioformats,ometiff,openjpeg,openslide,tiff]' 'scikit_image' --find-links https://girder.github.io/large_image_wheels
+          pip install --upgrade 'large-image[bioformats,ometiff,openjpeg,openslide,tiff]' 'scikit_image' --find-links https://girder.github.io/large_image_wheels
+          pip install --upgrade typing-extensions
```

Comment:
Probably if you moved this after installing tensorflow it would work on Python 3.8.

Reply:
Yes, that works.